include/unifex/any_unique.hpp

```diff
@@ -334,71 +360,36 @@ class _byval<CPOs...>::type
 
   template <typename Concrete, typename... Args>
   explicit type(std::in_place_type_t<Concrete> tag, Args&&... args)
-    : type(
-          std::allocator_arg,
-          std::allocator<unsigned char>{},
-          tag,
-          (Args &&) args...) {}
+    : impl_(new Concrete((Args&&) args...))
+    , vtable_(vtable_holder_t::template create<Concrete>()) {}
```

Comment:
Ah, I see you're bypassing the allocator version here.

Reply:
Yep.